$NetBSD: patch-libguestlib_Makefile.am,v 1.4 2024/04/09 01:12:36 pho Exp $

Do not link against dl. It is already handled by ./configure, and doing it
explicitly here causes a linkage failure on platforms where libdl doesn't
exist (such as NetBSD).

--- libguestlib/Makefile.am.orig	2024-02-06 15:07:33.000000000 +0000
+++ libguestlib/Makefile.am
@@ -50,7 +50,7 @@ libguestlib_la_SOURCES += $(libguestlib_
 libguestlib_la_SOURCES += $(libguestlib_rpcchanneldir)/simpleSocket.c
 endif
 
-libguestlib_la_LIBADD += -ldl -lrt
+libguestlib_la_LIBADD += -lrt
 # We require GCC, so we're fine passing compiler-specific flags.
 # Needed for OS's that don't link shared libraries against libc by default, e.g. FreeBSD
 libguestlib_la_LIBADD += -lc
